packages/docs/src/components/banner.js

```diff
@@ -39,34 +34,45 @@ export default props =>
           borderRadius: 6,
           ':hover': {
             color: 'background',
-            bg: 'text',
+            bg: 'text'
           }
-        },
+        }
       }
     }}
   >
-    <Box
-      py={[ 5, 6 ]}
-      color='background'
-      bg='primary'>
+    <Box color="background" bg="primary">
       <Container
         css={{
-          display: 'flex',
-          flexDirection: [ 'column', 'column', 'row' ],
-          alignItems: [ 'flex-start', 'flex-start', 'center' ],
-          justifyContent: 'space-between',
-        }}>
-        <Logo
-          size='1em'
-          color='currentcolor'
+          py: [5, 6]
+        }}
+      >
+        <Box
           css={{
-            fontSize: [ 96, 96, 160 ],
+            display: 'grid',
+            gridTemplateColumns: ['1f', 'repeat(3, 1fr)'],
```

Comment:
I feel like this whole banner layout could be improved up wrt how the logo is placed, etc. so feel free to be creative with the layout inside the banner